packages/jaeger-ui/src/utils/ts/pluckTruthy.tsx

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 // Because TypeScript doesn't believe in Array#filter(Boolean)
-export default function pluckTruthy<T>(values: (T | any)[]): T[] {
+export default function pluckTruthy<T>(values: (T | undefined)[]): T[] {
```

Comment:
this change doesn't make sense, `any` actually seems reasonable here.

Reply:
ok, I will revert back it to any.

Reply:
just add linter-ignore for that line